parcels/field.py

```diff
@@ -1753,6 +1757,13 @@ def __init__(self, filename, dimensions, indices, netcdf_engine, timestamp=None,
         self.rechunk_callback_fields = rechunk_callback_fields
         self.chunking_finalized = False
         self.lock_file = lock_file
+        if "name_maps" in kwargs.keys() and kwargs["name_maps"] is not None and isinstance(kwargs["name_maps"], dict):
+            self._name_maps = kwargs["name_maps"]
+            # ==== code for merging instead of replacing the map - choose which may be most applicable ==== #
```

Comment:
I'd say merging is better, also for when users have some pre-built and some non-standard chunking dimensions

Reply:
agreed.